tests/core/pyspec/eth2spec/test/helpers/deposits.py

```diff
@@ -320,14 +352,21 @@ def run_deposit_processing_with_specific_fork_version(
     yield from run_deposit_processing(spec, state, deposit, validator_index, valid=valid, effective=effective)
 
 
-def run_deposit_request_processing(spec, state, deposit_request, validator_index, valid=True, effective=True):
+def run_deposit_request_processing(
+        spec,
+        state,
+        deposit_request,
+        validator_index,
+        effective=True):
+
     """
     Run ``process_deposit_request``, yielding:
       - pre-state ('pre')
       - deposit_request ('deposit_request')
       - post-state ('post').
-    If ``valid == False``, run expecting ``AssertionError``
     """
+    assert is_post_electra(spec)
```

Comment:
is it needed?

Reply:
Might be not as these methods are called from post electra tests only